src/st/no2/352B4.c

```diff
@@ -1,7 +1,28 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 #include "no2.h"
 
-INCLUDE_ASM("st/no2/nonmatchings/352B4", func_us_801B52B4);
+u8 D_us_80180D38[] = {
```

Comment:
Nitpick but generally we prefer to make data imports `static` where possible if they are not used outside this file.